cosmos-init/plugins/packages/cosmosc2-demo/Rakefile

```diff
@@ -36,5 +36,7 @@ task :build => [:require_version] do
   if platform == 'mswin32' or platform == 'mingw32'
     puts "Warning: Building gem on Windows will lose file permissions"
   end
+  # Build the widgets in the src directory
+  system("yarn run build")
```

Comment:
I can see that we're already doing this via the `docker-package-build` script but that only happens in the cosmos-init Dockerfile. As an example for other plugins, I think this command should also be here.